tests/manage/z_cluster/test_mon_disk_low.py

```diff
@@ -0,0 +1,80 @@
+"""
+Testing MON disk low threshold
+Automating BZ#1964055
+"""
+import logging
+import time
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, tier2, bugzilla, skipif_ocs_version
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility import utils
+
+log = logging.getLogger(__name__)
+
+MON_POD = pod.get_mon_pods()[0]
+MON_SUFFIX = MON_POD.get().get("metadata").get("labels").get("mon")
+WORKLOADS_DIR = f"/var/lib/ceph/mon/ceph-{MON_SUFFIX}/workloads"
+SLEEP_TIMEOUT = 40
+
+
+@pytest.fixture(autouse=True)
+def workloads_dir_setup(request):
+    """
+    Setting up the environment for the test
+    """
+
+    def finalizer():
+        MON_POD.exec_cmd_on_pod(f"rm -rf {WORKLOADS_DIR}")
+        time.sleep(SLEEP_TIMEOUT)
+        utils.ceph_health_check()
+
+    log.info(f"Selected mon '{MON_POD.name}'")
+    MON_POD.exec_cmd_on_pod(f"mkdir {WORKLOADS_DIR}")
+    request.addfinalizer(finalizer)
+
+
+@tier2
+@bugzilla("1938049")
+@skipif_ocs_version("<4.9")
+class TestMonDiskLow(E2ETest):
+    """
+    Testing MON disk low threshold
+    """
+
+    def exec_dd_cmd(self, first=True):
+        """
+        Executes for loop for dd command
+        First loop for reaching ~80% of mon disk
+        Second loop for reaching >80% of mon disk
+        """
+        loop_cnt = "1..8000"
+        # write_cmd = "for i in {}; do dd if=/dev/urandom "
+        if not first:
+            loop_cnt = "8001..8500"
+            # write_cmd = "for i in {8001..8500}; do dd if=/dev/urandom "
+
+        write_cmd = f"for i in {{{loop_cnt}}}; do dd if=/dev/urandom "
+        write_cmd = (
+            write_cmd
+            + f"of=/var/lib/ceph/mon/ceph-{MON_SUFFIX}/workloads/file$i bs=5M count=1; done"
+        )
+        MON_POD.exec_sh_cmd_on_pod(command=write_cmd, sh="sh")
+        time.sleep(SLEEP_TIMEOUT)
+
+    def test_mon_disk_low(self):
+        """
+        Creating a Ceph Filesystem
+        """
+        # write until ~80%
```

Comment:
how are we making sure ~80% usage is reached?

Reply:
Test changed and it checks the used percantage after every dd command